HelloMixpanel/HelloMixpanelTests/Mixpanel_Testing.h

```diff
@@ -0,0 +1,80 @@
+//
+//  Mixpanel_Testing.h
```

Comment:
Would it be possible to call it MixpanelTesting? That way we are keeping the same naming convention we currently have